var/spack/repos/builtin/packages/graphviz/package.py

```diff
@@ -45,6 +39,8 @@ class Graphviz(AutotoolsPackage):
                 'bindings'.format(lang))
 
     # Feature variants
+    variant('doc', default=False,
```

Comment:
Opinions on doc vs docs? I can't remember which is more common.

Reply:
Looks like `doc`:
```sh
$ for w in doc docs; do echo "$w: $(grep "variant('${w}'" */package.py | wc -l)"; done
doc:       34
docs:       20
```

Reply:
At some point we should make these consistent and add it to https://spack.readthedocs.io/en/latest/packaging_guide.html#variant-names, but that is for another PR.